### DISCUSSION TITLE
Playlist files numbering option

### DISCUSSION THREAD

**Discussion by EvanHant** [shaked6540/YoutubePlaylistDownloader#77] [Ideas]
Hello kind sir! Liked your little app at first fight ) 

I have a suggestion for playlists downloading and overall files naming: would be awesome if users will have ability to add prefix to file name with a few options

1. [Video name].[extension] (Default)
2. [Video serial number in playlist in format 001] [Video name].[extension]
3. [Video upload date in format yyyymmdd] [Video name].[extension]
Or even have a field for saved file mask with some of above tag variables.

To keep sequence and catalogisation nice and clean ) Thanks!

Cheers, Evan

**Comment by shaked6540** [comment]
I'll think about it, I don't have much time but it doesn't look like a big feature so I might add it soon :)

**Reply by ghost** [reply]
did you add the feature? if yes then can you tell me how to use it please??

**Reply by shaked6540** [reply]
Yes, check out "file name pattern" in the options. If you want to know how to use it look in the Help->File name pattern

**Reply by ghost** [reply]
thank you
this is a great app
thanks for making it
